src/main/java/org/spongepowered/api/data/Keys.java

```diff
@@ -3645,6 +3648,53 @@ public final class Keys {
      */
     public static final Key<Value<WorldTypeEffect>> WORLD_TYPE_EFFECT = Keys.key(ResourceKey.sponge("world_type_effect"), WorldTypeEffect.class);
 
+    /**
+     * The {@link #ITEM_DURABILITY} damage an {@link ItemStack} takes per attack.
+     */
+    public static final Key<Value<Integer>> WEAPON_DAMAGE_PER_ATTACK = Keys.key(ResourceKey.sponge("weapon_damage_per_attack"), Integer.class);
+
+    /**
+     * The amount of {@link Ticks} this {@link ItemStack} disables blocking for on successful attack.
+     */
+    public static final Key<Value<Ticks>> DISABLE_BLOCKING_TICKS = Keys.key(ResourceKey.sponge("disable_blocking_ticks"), Ticks.class);
+
+    /**
+     * The amount of {@link Ticks} player must use this {@link ItemStack} for to block attacks successfully.
+     */
+    public static final Key<Value<Ticks>> BLOCK_DELAY_TICKS = Keys.key(ResourceKey.sponge("block_delay_ticks"), Ticks.class);
+
+    /**
+     * Multiplier applied to the cooldown during which blocking using this item is disabled.
+     *
+     * @see <a href="https://minecraft.wiki/w/Data_component_format#blocks_attacks">blocks_attacks</a>
+     */
+    public static final Key<Value<Float>> DISABLED_BLOCKING_COOLDOWN_SCALE = Keys.key(ResourceKey.sponge("disabled_blocking_cooldown_scale"), Float.class);
```

Comment:
```suggestion
    public static final Key<Value<Double>> DISABLED_BLOCKING_COOLDOWN_SCALE = Keys.key(ResourceKey.sponge("disabled_blocking_cooldown_scale"), Double.class);
```
We generally favor Double as thats forward compatible in case the underlying type changes.

Reply:
Maybe also rename to DISABLE_BLOCKING_TICKS_SCALE?